components/list/test/list.visual-diff.html

```diff
@@ -208,7 +236,7 @@
 		<div style="width: 400px;">
 			<div class="visual-diff" id="selectableHeader">
 				<d2l-list>
-					<d2l-list-header slot="header"></d2l-list-header>
+					<d2l-list-header slot="header" no-sticky></d2l-list-header>
```

Comment:
Lots of tests get kinda messed up if the headers are allowed to sticky on these fixtures.  So, we opt out of sticking for these tests so they do not break other tests.